### ISSUE TITLE
Does it make sense to use doctr to push with .nojekyll

### ISSUE THREAD

**Issue by ocefpaf** [drdoctr/doctr#131]
Right now `doctr` hardcodes a no `.nojekyll` as the design is to upload HTML pages to `gh-pages`. But what if one wants to use `doctr` to manage a page but still wants let GitHub build it? Should the `.nojekyll` be an option?

**Comment by gforsyth** [comment]
Hey @ocefpaf, yes, I think that makes sense to make it an option.  We hard-coded it to match our initial use-cases, but we're moving beyond those now.  I think I would still like to have the `.nojekyll` put there by default, but it should be an option to disable it.  I'm open to suggestions/differences of opinion on this, though.

**Comment by ocefpaf** [comment]
> I think I would still like to have the .nojekyll put there by `default`

:+1:

**Comment by asmeurer** [comment]
I believe if only happens once, when doctr creates gh-pages if it doesn't exist. So you can manually remove it if you don't want it. An option seems like overkill, since it would be ignored after the first run. Maybe we just need some documentation.

**Comment by ocefpaf** [comment]
I agree that an option is an overkill but removing it afterwards seems like a hack. Maybe this could be something that happens at `doctr configure` time.

**Comment by asmeurer** [comment]
It's not a hack. Doctr is only in charge of the directory it deploys to. You are free to modify gh-pages outside that directory however you want (or more precisely, it is in charge of the files it deploys, so even if the directory is `.` it doesn't care about `.nojekyll` unless that is one of the files it deploys).
